clients/geth/specular/rollup/services/sync.go

```diff
@@ -126,7 +135,7 @@ func (b *BaseService) SyncInbox(start, end uint64) error {
 	log.Info("Syncing inbox", "start", start, "end", end)
 	abi, err := bindings.ISequencerInboxMetaData.GetAbi()
 	if err != nil {
-		log.Crit("Failed to get ISequencerInbox ABI", "err", err)
+		return fmt.Errorf("["+funcName()+"] Failed to get ISequencerInbox ABI, err: %w", err)
```

Comment:
use %s instead of concat

also maybe lets define a function that abstracts this away too—e.g. `CustomErrorf` and forward these arguments into `fmt.Errorf`

Reply:
Logs updated as per geth logs